addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -30,8 +30,11 @@
 <t t-name="mail.messaging_menu.preview" owl="1">
     <t t-set="thread" t-value="messaging.threads[preview.id]"/>
     <div class="o-mail-notification-item d-flex cursor-pointer border-bottom align-items-center" t-on-click="() => this.openDiscussion(thread.id)">
-        <div class="m-1" style="width:40px;height:40px;">
+        <div class="position-relative d-flex m-1" style="width:40px;height:40px;">
             <img class="w-100 h-100 rounded-circle" alt="Thread Image" t-att-src="thread.imgUrl" />
+            <div class="d-flex justify-content-center flex-shrink-0 position-absolute bottom-0 end-0 align-items-center rounded-circle bg-100 small o-muted">
+                <PartnerImStatus t-if="thread.type === 'chat'" partner="messaging.partners[thread.chatPartnerId]"/>
```

Comment:
The `t-if` should be moved to `div` above, as you don't want to show `rounded-circle` neither when there's no `im_status` to show